py_gtfs_rt_ingestion/ingest.py

```diff
@@ -8,8 +8,11 @@
 import sys
 
 from multiprocessing import Pool
+from typing import NamedTuple
```

Comment:
i hadn't seen this type before. seems like a good thing to have around.

Reply:
Usually this NamedTuple or dataclass is used for this kind of small data structure.